package.json

```diff
@@ -11,7 +11,11 @@
   },
   "license": "Apache-2.0",
   "files": [],
+  "bin": {
+    "matrix-gen-i18n": "scripts/gen-i18n.js"
+  },
   "scripts": {
+    "i18n": "matrix-gen-i18n",
```

Comment:
probably need a matching  `diff-i18n` for CI and `prunei18n`

Reply:
Adding this to the list of tasks to do before merging this PR 👍